pybamm/models/submodels/interface/sei/ec_reaction_limited.py

```diff
@@ -131,3 +111,48 @@ def set_initial_conditions(self, variables):
         j_sei_0 = pybamm.Scalar(0)
 
         self.initial_conditions = {L_sei: L_sei_0, j_sei: j_sei_0}
+
+    def _get_standard_concentraion_variables(self, variables):
```

Comment:
If this function is only ever called from `get_coupled_variables`, there is no need to make it a separate function. Ideally, we should find a way to make it use the method from `base_sei` (e.g. with inner set to 0 and outer representing the total)